packages/@okta/vuepress-theme-prose/assets/css/okta/pages/multi-tenancy.scss

```diff
@@ -0,0 +1,54 @@
+.multi-tenancy {
+    //Image format
+    .image-at-100 {
+        text-align: center;
+
+        img {
+            max-width: 100%;
+        }
+    }
+    .image-at-90 {
+        text-align: center;
+
+        img {
+            max-width: 90%;
+        }
+    }
+    .image-at-80 {
+        text-align: center;
+
+        img {
+            max-width: 80%;
+        }
+    }
+    .image-at-70 {
+        text-align: center;
+
+        img {
+            max-width: 70%;
+        }
+    }
+    .image-at-60 {
+        text-align: center;
+
+        img {
+            max-width: 60%;
+        }
+    }
+    .image-at-50 {
+        text-align: center;
+
+        img {
+            max-width: 50%;
+        }
+    }
+    //Lists
+    .heading-2-subtext {
```

Comment:
I'll defer to Christine on the approval of specific styles.

Reply:
I'll coordinate with Christine on these styles and will make additional changes in the next PR if necessary (Another PR is planned for the Multi-tenancy documentation to incorporate new content feedback.